sample-libs/Samples.DatabaseHelper/DapperTestHarness.cs

```diff
@@ -60,6 +61,12 @@ public async Task RunAsync()
                 }
             }
 
+            var t1 = CallQueryAsync(_connection, SelectOneCommandText);
```

Comment:
You should be able to `await` both `CallQueryAsync` calls and then you can get rid of the additional `Task.Delay` call

Reply:
Of course, what was I thinking!